.idea/runConfigurations/Flutter_Plugin__tests_.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
I think IntelliJ does that to escape away any chars that might be dodgy on the file system. In this case, the name of the launch config in the UI is `'Flutter Plugin (tests'`.